bastion/src/dispatcher.rs

```diff
@@ -55,11 +58,43 @@ pub enum DispatcherType {
     Named(String),
 }
 
-#[derive(Debug)]
-/// The special handler, which is creating by default. Initially
-/// doesn't do any useful processing.
-pub struct DefaultDispatcherHandler;
+/// The default handler, which does round-robin.
+pub type DefaultDispatcherHandler = RoundRobinHandler;
+
+// Dispatcher that will do simple round-robin distribution
+#[derive(Default, Debug)]
+pub struct RoundRobinHandler {
+    index: AtomicU64,
+}
+
+impl DispatcherHandler for RoundRobinHandler {
+    // Will left this implementation as empty.
+    fn notify(
```

Comment:
any idea about what is expected here ?

Reply:
This will either register the child or remove the child on join or on leave.

Reply:
Currently the default handler doesn't provide any specific implementation for it. I decided to left it to developers about how to react on those things. So, just left it as-is